doc/quick-start-user-guide.md

```diff
@@ -44,14 +35,15 @@ We recommend that you use the RPMs made available nightly as they contain the la
 # yum install glusterd2
 ```
 
-Alternatlively, if you are using a non-RPM based distro, you can download
-binaries of the latest release. Like all Go programs, glusterd2 is a single
-binary (statically linked) without external dependencies. You can download the
-[latest release](https://github.com/gluster/glusterd2/releases) from Github.
+For other distros, you can download binaries of the [latest release](https://github.com/gluster/glusterd2/releases) from Github. 
+Like all Go programs, glusterd2 is a single binary (statically linked) without 
+external dependencies.
 
-### Running glusterd2
+**Config File:** Default path of glusterd2 config file is `/etc/glusterd2/glusterd2.toml`.
 
-**Create a working directory:** This is where glusterd2 will store all data which includes logs, pid files, etcd information etc. For this example, we will be using a temporary path. If a working directory is not specified, it defaults to current directory.
+### Custom configuration for glusterd2 [Optional].
+
+**Create a working directory:** This is where glusterd2 will store all data which includes logs, pid files, etcd information etc. For the purposes of this example, we will be using a temporary path. If a working directory is not specified, it defaults to current directory.
```

Comment:
There's no such thing as working directory anymore. The location of logs and pid files can be different.